.github/install_bazel.sh

```diff
@@ -0,0 +1,13 @@
+if ! bazel version; then
```

Comment:
https://github.com/google/benchmark/blob/main/.github/workflows/bazel.yml assumes it's available and it seems to be.

why do we need this?

Reply:
Linux wheel build commands run inside a Docker container, which does not have access to the runner's installed tools.

Reply:
As for why the script checks for the install even if it is never available on vanilla manylinux images: Other emulated Linux architectures (ppc64le etc.) do not have Bazel binaries ready for download, and would have to be bootstrapped (generally at image build time).
If we wanted to build ppc, we would probably use a custom manylinux-based Docker image with Bazel available - and the install check would then decay gracefully into an immediate zero exit.